finetune/encoding/target_encoders.py

```diff
@@ -222,7 +222,8 @@ def transform(self, out, labels):
         labels_out = [pad_idx for _ in out.tokens]
         for label in labels:
             for i, (start, end, text) in enumerate(zip(out.token_starts, out.token_ends, out.tokens)):
-                if end > label["end"]:
+                # Label extends less than halfway through token
+                if label["end"] < (start + end + 1) // 2:
```

Comment:
+1 to ensure labeling exactly half of a token causes a positive label.  Kinda weird.